embedx/config.schema.json

```
@@ -1350,6 +1350,16 @@
         "admin": {
           "type": "object",
           "properties": {
+            "access_log": {
```

```suggestion
            "request_log": {
```
I think request log makes more sense than access log.

---

And it would be good to add a description here.

---

Fixed in last commit.